packages/vitaminjs-jest/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+  "presets": [
+    ["env",
+      {
+        "targets": {
+          "node": "current"
+        }
+      }
+    ],
+    "stage-1"
+  ]
+}
```

Comment:
Do we need this configuration file? We already have `babelrcJestTransformer.js`.